diff --git a/node_modules/flowbite-react/dist/helpers/get-tailwind-version.js b/node_modules/flowbite-react/dist/helpers/get-tailwind-version.js
index 88846f2..b3357aa 100644
--- a/node_modules/flowbite-react/dist/helpers/get-tailwind-version.js
+++ b/node_modules/flowbite-react/dist/helpers/get-tailwind-version.js
@@ -1,4 +1,5 @@
-import version from 'tailwindcss/version.js';
+import pkg from 'tailwindcss/package.json';
+const version = pkg.version;
 
 function getTailwindVersion() {
   try {
